src/main/kotlin/org/genspectrum/ingest/proc/JoinSC2NextstrainOpenData.kt

```diff
@@ -175,7 +184,7 @@ private fun clean(entry: MutableEntry) {
     entry.apply {
         renameMetadata(oldToNewMetadataNames)
         selectMetadata(selectedMetadata)
-        mapToNull()
+        mapToNull(metadataFieldsMapToNull)
```

Comment:
Ah, I didn't review carefully enough and forgot how `mapToNull()` works: this is not the right implementation if we take a look at the code of [mapToNull()](https://github.com/GenSpectrum/ingest/blob/main/src/main/kotlin/org/genspectrum/ingest/entry/mapToNull.kt#L7). I'll fix it.